docs/reference/training/experiment-config-reference.rst

```diff
@@ -1454,6 +1454,8 @@ GPU type with:
    slurm:
       gpu_type: tesla
 
+.. _sbatch-args:
```

Comment:
What is this?

Reply:
A label that we can cross reference to this section.